lib/net/networking_engine.js

```diff
@@ -46,9 +46,9 @@ goog.require('shaka.util.Timer');
  */
 shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
   /**
-   * @param {function(number, number)=} onProgressUpdated Called when a progress
-   *   event is triggered. Passed the duration, in milliseconds, that the
-   *   request took, and the number of bytes transferred.
+   * @param {function(number, number, boolean)=} onProgressUpdated Called when
+   *   a progress event is triggered. Passed the duration, in milliseconds,
+   *   that the request took, and the number of bytes transferred.
```

Comment:
This should list the new third parameter (allowSwitch). Right now it's only listing timeMs and byteLength.

Reply:
Done!